gum/gum.c

```diff
@@ -506,6 +509,18 @@ gum_on_log_message (const gchar * log_domain,
 
     return;
   }
+#else
```

Comment:
I'd say the stdio fallback doesn't make any sense on Windows – it only does on i/macOS where some processes may not have `CoreFoundation` loaded. So we should move this to the top and do like:

```C
#if defined (HAVE_WINDOWS)
...
#elif defined (HAVE_ANDROID)
...
#else
# ifdef HAVE_DARWIN
```